src/site/content/en/blog/url/index.md

```diff
@@ -0,0 +1,296 @@
+---  
+title: What are the parts of a URL?  
+subhead: "What's the difference between a domain name, hostname, site and origin? This article explains."
+authors:  
+  - samdutton  
+date: 2023-02-22
+hero: image/80mq7dk16vVEg8BBhsVe42n6zn82/EAQ6QLeBBonbpyqeGWPb.jpg
+alt:   
+tags:  
+    - blog
+    - html
+    - network
+    - privacy
+---
+
+Most of the time it's fine to be relaxed about naming websites and URLs. It's
+usually OK to say things like "I bought a domain name" or "Our images are hosted on a different
+site," even if that's not strictly true.
+
+However, in some contexts it's crucial to be accurate and specific when referring to the parts of a
+URL. For example, when dealing with cookies, you need to understand the difference between
+[site](#site) and [origin](#origin). This article is designed to help with that.
+
+The explanations here are only intended as a guide:
+[The URL Living Standard](https://url.spec.whatwg.org/) provides formal definitions of URL parts.
+
+{% Aside %}
+The named parts of a URL may coincide, but that doesn't mean they're equivalent!   
+
+The [FQDN](#fqdn), [eTLD+1](#etld1), [hostname](#hostname) and [registrable domain](#registrable-domain)
+may be the same for some URLs, but each term has a different meaning.
+{% endAside %}
+
+Edit the URL in the Glitch below to see the part names. (You can also open this in a separate tab at [url-parts.glitch.me](https://url-parts.glitch.me).)
```

Comment:
As above, I'd like to see this Glitch align with URL and use "registrable domain".

Reply:
I've updated the Glitch to use "registrable domain" (with eTLD+1 in brackets, in case people see that term).